### DISCUSSION TITLE
See the logs in Nginx proxy amanger?

### DISCUSSION THREAD

**Discussion by neutropico** [tteck/Proxmox#1187] [Help Needed]
Hi

Maybe its a dumb question... but where i can debug NPM? Im trying to locate the logs and it aparently doesnt log anything...

**Comment by tteck** [comment]
`/var/log/`

**Reply by neutropico** [reply]
But there there is no nginx.log or something like that... Maybe it should enable in somehow...?

`alternatives.log  apt  auth.log  btmp  daemon.log  dpkg.log  faillog  journal  lastlog  letsencrypt  mail.info  mail.log  private  runit  syslog  wtmp`
